package.json

```diff
@@ -2,18 +2,39 @@
   "name": "supertokens-auth-react",
   "version": "0.0.1",
   "description": "ReactJS SDK that provides login functionality with SuperTokens.",
-  "main": "index.js",
-  "dependencies": {},
+  "main": "./lib/build/index.js",
   "devDependencies": {
-    "isomorphic-fetch": "2.2.1",
+    "@babel/cli": "^7.12.1",
+    "@babel/core": "^7.12.1",
+    "@babel/preset-env": "^7.12.1",
+    "@babel/preset-react": "^7.12.1",
+    "@babel/preset-typescript": "^7.12.1",
+    "@babel/register": "^7.12.1",
+    "@types/node": "^14.11.10",
+    "@types/react": "^16.9.53",
+    "@types/react-router-dom": "^5.1.6",
+    "babel-plugin-transform-class-properties": "^6.24.1",
+    "jsdom": "16.4.0",
+    "jsdom-global": "3.0.2",
     "prettier": "1.18.2",
-    "typescript": "3.5.2",
-    "axios": "*"
+    "puppeteer": "^5.3.1",
+    "react": "^16.14.0",
+    "react-dom": "^16.0.0",
+    "react-router-dom": "^5.2.0",
+    "regenerator-runtime": "^0.13.7",
+    "typescript": "3.5.2"
+  },
+  "peerDependencies": {
+    "react": "^16.14.0",
+    "react-router-dom": "^5.2.0"
```

Comment:
peer dependency that is not required if you are not using the `getSuperTokensRoutesForReactRouterDom component`. Peer dependencies  versions discussed here https://github.com/supertokens/supertokens-auth-react/issues/14